jakarta.ls/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.11</version>
+      <version>4.13.2</version>
```

Comment:
You may also need to change: [jakarta.eclipse/org.eclipse.lsp4jakarta.lsp4e.test/META-INF/MANIFEST.MF](https://github.com/eclipse/lsp4jakarta/blob/32886c9a25b4deb01fe8bd4d0c9545f082ef4cec/jakarta.eclipse/org.eclipse.lsp4jakarta.lsp4e.test/META-INF/MANIFEST.MF#L8)